### DISCUSSION TITLE
wiring diagram isn't available

### DISCUSSION THREAD

**Discussion by YAYASIMPSON** [softwarecrash/Solar2MQTT-ESP8266#61] [General]
since many day i saw jpg of wiring diagram are disapear can you resend for me ? thanks

**Comment by all-solutions** [comment]
This is currently a very strange bug on Github... 
Now try again, I just updated the page (again).

**Reply by YAYASIMPSON** [reply]
thank's it's good

**Comment by haadi76** [comment]
Don't you think the wiring diagram is wrong. As in orignal Solar2MQTT alpha the esp was connected to ttl side of the max3232 circuit. But in the wiring diagram rs232 side is connected to the esp8266

**Comment by YAYASIMPSON** [comment]
Hello ,

is it possible to create a new version for POWMR with this yaml
explanation, you can see it's on RS232 TTL dialogue .
https://github.com/odya/esphome-powmr-hybrid-inverter
because your technic is available and simply to use but version of Odya use
ESPHOME and it's bored to use it.



Le jeu. 30 nov. 2023 à 05:28, haadi76 ***@***.***> a écrit :

> Don't you think the wiring diagram is wrong. As in orignal Solar2MQTT
> alpha the esp was connected to ttl side of the max3232 circuit. But in the
> wiring diagram rs232 side is connected to the esp8266
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/softwarecrash/Solar2MQTT/discussions/61#discussioncomment-7713085>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AW56KNO2PFI4ZEO3AW362CDYHADN5AVCNFSM6AAAAABAAEIHK2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TOMJTGA4DK>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤
Yannick CHALOT
52 Sente du Pré, 95320 Saint leu la forêt
tel : 06.08.71.05.36
@ : ***@***.***
¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤¤

**Reply by softwarecrash** [reply]
Without the Protokoll No way.

Looks Like it is a modbus Inverter, that actual Not supported by my firmware
